modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -649,8 +649,25 @@ properties:
 
               As with the SDK, when using the `couchbase://` or `couchbases://` schemes, the port is not required, but if specified should be the external/internal bucket ports (defaults are 11210 or 11207 respectively). Attempting to use the admin ports (8091/18091) will result in a startup error.
 
+              *Alternate Addresses*
+
               On startup, Sync Gateway will try each hostname that is provided until it is able to connect successfully.
 
+              By default, if a remote cluster has an external address set, then when SG connects it will always use the external address.
+              However, it is possible to override this behavior by adding a `network` parameter to the connection string.
```

Comment:
Unfortunately it's not this simple for the `auto` case (SG's default) - We apply a heuristic to determine whether to choose between external or default (internal).

This is based on the hostnames supplied in the connection string: SG will use external networking if the hostname doesn't match Couchbase Server's internal node addresses, and there's an external address defined. Failing that, default (internal) networking is used.

Adding to the confusion, the SDK/SG default of `auto` is not the same as `network=default`!  But that's outside of our control :( 
So need to be very selective when using either of those words to avoid misinterpretation.

Reply:
Ok. Cheers. Will re-work this tomorrow. Thanks @bbrks

Reply:
Does this work better @bbrks ?

`By default, if a remote cluster has an external address set, then when SG connects it will apply a heuristic to determine whether to choose between external or default (internal).

The choice is based on the hostname supplied in the connection string. 
 - SG uses external networking only when the supplied hostname doesn't match any of Couchbase Server's internal node addresses, and an external address is defined. 
- In all other cases Sync Gateway uses the default (internal) networking.
              
However, it is possible to override this behavior by adding a `network` parameter to the connection string.

The `network` parameter can be --
- auto -- this is the default value if no parameter is provided. In this case the heuristic described above is applied to determine the address used; so effectively there is no override.
- external -- to always force use of the external address
- default -- to always force use of the internal address`

Reply:
Yes, this sounds good. Better than I would have been able to phrase it! 😄

Reply:
Cheers. It's good to go then.